source/_docs/modules.md

```diff
@@ -39,6 +39,71 @@ This module provides general methods your site needs to access aspects of the in
 
 Note: Currently, there is no API module available for Drupal 8.
 
+Issue: For sites with Filefield Paths module enabled and a lot of image styles, some transactions may hit some performance issues as image paths are flushed due to its volume. 
+
+Workaround: Add the image path files clearing execution using the Drupal Queue API and be executed during cron run
+
+```
+function image_path_flush($path) {
+  // Following pantheon function isn't even used and subverts the fix.
+  // Pantheon optimization: one bulk operation is better.
+  // if (function_exists('pantheon_api_image_path_flush') && function_exists('pantheon_bulk_file_delete')) {
+  //  pantheon_api_image_path_flush($path);
+  //  return;
+  //}
+  $styles = image_styles();
+  foreach ($styles as $style) {
+    $image_path = image_style_path($style['name'], $path);
+    if (defined('PANTHEON_ENVIRONMENT')) {
```

Comment:
I think it makes sense to exclude LocalDev/Lando environments here (i.e.  `&& PANTHEON_ENVIRONMENT != 'lando'`). Assuming that the performance problem is specific to Pantheon's filesystem, then it doesn't make sense to change the behavior on local environments that don't share that problem. Also, developers often don't have cron set up locally the same way it is configured in the cloud so, as written, this workaround could result in the queue rarely being processed locally.